lms/static/teams

```diff
@@ -0,0 +1 @@
+../djangoapps/teams/static/teams
```

Comment:
I'm guessing the necessity of adding this filesystem link is going to be explained in the documentation?